lib/pipeline/pipeline.js

```diff
@@ -75,95 +76,86 @@ class Pipeline {
                 if(idx < contracts.length - 1) importsHelperFile.write(',\n'); // add a comma if we have more contracts to add
               });
             }, function(){
-              importsHelperFile.write('\n}'); // close the module.exports = {} 
+              importsHelperFile.write('\n}'); // close the module.exports = {}
               importsHelperFile.close(next); // close the write stream
             });
           });
         });
       },
+      function runWebpack(next) {
+        self.logger.info(__(`running webpack with '${self.webpackConfigName}' config...`));
+        Object.keys(self.assetFiles)
+          .filter(key => key.match(/\.js?$/))
+          .forEach(key => {
+            self.logger.info(__("writing file") + " " + (utils.joinPath(self.buildDir, key)).bold.dim);
+          });
+        let built = false;
+        const webpackProcess = new ProcessLauncher({
+          modulePath: utils.joinPath(__dirname, 'webpackProcess.js'),
+          logger: self.logger,
+          events: self.events,
+          exitCallback: function (code) {
+            if (!built) {
+              return next(`Webpack build exited with code ${code} before the process finished`);
+            }
+            if (code) {
+              self.logger.error(__('Webpack build process exited with code ', code));
+            }
+          }
+        });
+        webpackProcess.send({action: constants.pipeline.init, options: {webpackConfigName: self.webpackConfigName}});
+        webpackProcess.send({action: constants.pipeline.build, assets: self.assetFiles, importsList});
+
+        webpackProcess.once('result', constants.pipeline.built, (msg) => {
+          built = true;
+          webpackProcess.kill();
```

Comment:
It would be better to have webpack be a long running proces, in dev server mode, but that's going to require careful thinking about, and changes to, the overall build system.